test/fixtures/keys/selfsigned-no-keycertsign/https_renew_cert.sh

```diff
@@ -1,6 +1,12 @@
-#!/bin/bash
-openssl genrsa -out rsa.pem 2048
-openssl rsa -in rsa.pem -out key.pem
-openssl req -sha256 -new -key key.pem -out csr.pem -subj "/CN=localhost"
-openssl x509 -req -extfile cert.conf -extensions v3_req -days 365 -in csr.pem -signkey key.pem -out cert.pem
+#!/usr/bin/env bash
```

Comment:
I don't think we need bash at all, do we?
```suggestion
#!/bin/sh
```

Reply:
Does `sh` do more reliable error handling than bash? I don't think `sh` supports `pipefail` etc.

Reply:
Ah right, there are workarounds (https://stackoverflow.com/questions/13084352/how-to-implement-set-o-pipefail-in-a-posix-way-almost-done-expert-help-nee) but it's probably not worth it indeed.